central/rhsso/load_secret.go

```diff
@@ -0,0 +1,18 @@
+package rhsso
+
+import (
+	"io"
+	"os"
+)
+
+func LoadRhSsoSecret() string {
```

Comment:
I don't think we should have such functions in the code. Instead, we need to reference a secret in the auth provider config and load the secret dynamically.